static/app/gettingStartedDocs/node/serverlesscloud.tsx

```diff
@@ -0,0 +1,232 @@
+              {code: <code />}
+            )}
+          </p>
+        ),
+        code: `
+        api.get("/debug-sentry", function mainHandler(req, res) {
+          throw new Error("My first Sentry error!");
+        });
+    `,
+      },
+      {
+        language: 'javascript',
+        description: (
+          <p>
+            {tct(
+              '[code:requestHandler] accepts some options that let you decide what data should be included in the event sent to Sentry. Possible options are:',
+              {code: <code />}
+            )}
+          </p>
+        ),
+        code: `
+// keys to be extracted from req
+request?: boolean | string[]; // default: true = ['cookies', 'data', 'headers', 'method', 'query_string', 'url']
+
+// server name
+serverName?: boolean; // default: true
+
+// generate transaction name
+//   path == request.path (eg. "/foo")
+//   methodPath == request.method + request.path (eg. "GET|/foo")
+//   handler == function name (eg. "fooHandler")
+transaction?: boolean | 'path' | 'methodPath' | 'handler'; // default: true = 'methodPath'
+
+// keys to be extracted from req.user
+user?: boolean | string[]; // default: true = ['id', 'username', 'email']
+
+// client ip address
+ip?: boolean; // default: false
+
+// node version
+version?: boolean; // default: true
+
+// timeout for fatal route errors to be delivered
+flushTimeout?: number; // default: undefined
+    `,
+      },
+      {
+        language: 'javascript',
+        description: (
+          <p>
+            {tct(
+              'For example, if you want to skip the server name and add just user, you would use [code:requestHandler] like this:',
+              {code: <code />}
+            )}
+          </p>
+        ),
+        code: `
+        api.use(
+          Sentry.Handlers.requestHandler({
+            serverName: false,
+            user: ["email"],
+          })
+        );
+    `,
+      },
+      {
+        language: 'javascript',
+        description: (
+          <p>
+            {tct(
+              'By default, [code:errorHandler] will capture only errors with a status code of [code:500] or higher. If you want to change it, provide it with the [code:shouldHandleError] callback, which accepts middleware errors as its argument and decides, whether an error should be sent or not, by returning an appropriate boolean value.',
+              {code: <code />}
+            )}
+          </p>
+        ),
+        code: `
+        api.use(
+          Sentry.Handlers.errorHandler({
+            shouldHandleError(error) {
+              // Capture all 404 and 500 errors
+              if (error.status === 404 || error.status === 500) {
+                return true;
+              }
+              return false;
+            },
+          })
+        );
+    `,
+      },
+    ],
```

Comment:
IMO we don't necessarily need these steps as they're already very detailed and just list additional options to configure or use the SDK. I think it's fine if they are just mentioned in the docs and we make this page shorter.

Reply:
I'm aware that this is a change to the current onboarding wizard but I think it's for the better and it's essentially just removing stuff? but we can ofc revisit this later, too.